### PR TITLE
Add mongodb icon

### PR DESCRIPTION
**Pull request by sangdth**
Hello,

I have added `database.svg` icon, we can use it for many kinds of database in future, e.g. in MongoDB, I use its color. In future if we need SQL or anything, just copy it and map color.

Please let me know if there is anything need to be improved.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3582,6 +3573,15 @@
...
+      "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.0.3.tgz",
+      "integrity": "sha512-4AH6Z5fzNNBcH+6XDMfA/BTt87skxqJlO0lAh3Dker5zThcAxG6mKz+iGu308UKoPPQ8Dcqx/4JhujzltRa+hQ==",
+      "dev": true,
+      "requires": {
+        "safe-buffer": "5.1.1"
+      }
+    },
     "strip-ansi": {
       "version": "3.0.1",
       "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```
Comment (dfrankland) [inline package-lock.json]: No need to edit this file.

File: src/constants/mapColors.js [hunk 2]
```diff
@@ -6,11 +6,13 @@ export default {
...
+  mongod: '#589636',
+  mongo: '#589636',
   rails: '#CC0000',
   ruby: '#CC342D',
   shell: '#FFF',
   zsh: '#C5DB00',
   python: '#FFDF59',
   vim: '#007f00',
-  yarn: '#2C8EBB',
+  yarn: '#2C8EBB'
```
Comment (dfrankland) [inline src/constants/mapColors.js]: Please put back the `,`.

File: src/constants/mapIcons.js [hunk 3]
```diff
@@ -1,55 +1,19 @@
...
+  ],
+  http: ['wget', 'http'],
+  monitor: ['htop', 'iftop', 'top'],
+  database: ['mongodb', 'mongod', 'mongo', 'mysql'],
+  nodejs: ['node'],
+  ruby: ['irb', 'sidekiq', 'rake'],
+  shell: ['bash', 'fish', 'zsh'],
+  code: ['pico'],
+  read: ['tail', 'less', 'more']
 };
```
Comment (dfrankland) [inline src/constants/mapIcons.js]: Please lint this file, you can run `npm run lint` to find errors.

File: src/icons/index.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ export atom from './atom.svg';
 export bower from './bower.svg';
 export bq from './bq.svg';
 export clojure from './clojure.svg';
+export database from './database.svg';
```
Comment (dfrankland) [inline src/icons/index.js]: This file is sorted alphabetically. Can you put this `export` after the other `export`s that start with "c"?

**Review by dfrankland — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (sangdth) [reply package-lock.json]: No I did not, it was automatic and I hate that lock file too. I can't control it.

File: src/constants/mapColors.js (hunk 2 above)
Comment (sangdth) [reply src/constants/mapColors.js]: I will turn off the Prettier next time.

File: package-lock.json (hunk 1 above)
Comment (sangdth) [reply package-lock.json]: How could I avoid changing this lock file? I can not add it into .gitignore as this action will change your file, and everytime I run `npm install` it automatically edit it.

File: src/constants/mapIcons.js (hunk 3 above)
Comment (sangdth) [reply src/constants/mapIcons.js]: Done this.

File: src/icons/index.js (hunk 4 above)
Comment (sangdth) [reply src/icons/index.js]: Done this.

**Comment by sangdth** [comment]
Now I have to manually revert the package-lock.json because I don't know how to control it.

**Comment by sangdth** [comment]
Please accept this so I can add more icon.

File: package-lock.json [hunk 5]
```diff
@@ -3828,4 +3828,4 @@
       "dev": true
     }
   }
-}
```
Comment (dfrankland) [inline package-lock.json:3831]: Can you remove this change?

**Review by dfrankland — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 5 above)
Comment (sangdth) [reply package-lock.json:3831]: It's so weird.

First, the `package-lock.json` autogenerated and I don't know why it was different with your file. I commit it and pushed it here. Then you requested the change and I copied **yours** file and put it back.

**Pull request closed without merging**